app/system/database/seeds/DemoSchemaSeeder.php

```diff
@@ -97,17 +96,4 @@ protected function seedMenuItems()
             }
         }
     }
-
-    protected function seedCoupons()
-    {
-        if (DB::table('coupons')->count())
-            return;
-
-        DB::table('coupons')->insert($this->getSeedRecords('coupons'));
-    }
-
-    protected function getSeedRecords($name)
-    {
-        return json_decode(file_get_contents($this->recordsPath.'/'.$name.'.json'), TRUE);
-    }
```

Comment:
removing breaks other methods

Reply:
Sorry, too greedy with my deleting - didn't intend to delete

Reply:
No problem